scripts/build/check_init_priorities.py

```diff
@@ -280,7 +283,9 @@ def _parse_args(argv):
     parser.add_argument("-d", "--build-dir", default="build",
                         help="build directory to use")
     parser.add_argument("-v", "--verbose", action="store_true",
-                        help="enable verbose Output")
+                        help="enable verbose output")
+    parser.add_argument("-D", "--debug", action="store_true",
+                        help="enable debug output")
```

Comment:
Non-blocking:
`-v` and `-d` are mutually exclusive (sort of).   You should note that `--debug` implies `-v`.  You could also change `-v` to a count action (similar to twister).  So `-v` enables verbose and `-v -v` enables debug output.

Reply:
Good stuff, did not realize `action="count"` was a thing, switched to that, I prefer it as well. Thanks for the suggestion.